spring-cloud-gateway-core/src/test/java/org/springframework/cloud/gateway/filter/headers/ForwardedHeadersFilterTests.java

```diff
@@ -17,6 +17,9 @@
 
 package org.springframework.cloud.gateway.filter.headers;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
We have a certain way of ordering imports, static imports should be last.  Use the IDE settings files listed in the contributing guidelines to use the correct formatting guidelines

Reply:
Thx @ryanjbaxter , I forgot to disable my own code formatter settings.